script/dbdefs_to_js.pl

```diff
@@ -147,19 +162,17 @@ sub get_value {
         push @all_server_defs, $def;
         if ($CLIENT_DEFS{$def}) {
             $client_code .= $line;
+            my $browser_line  = "export const $def = DBDefs.$def;\n";
+            $browser_code .= $browser_line;
             push @all_client_defs, $def;
         }
     }
 }
 
-# Continue to allow importing as `import DBDefs from ...` by providing a
-# default export.
-$server_code .= 'export default {' . (join q(, ), @all_server_defs) . "};\n";
-$client_code .= 'export default {' . (join q(, ), @all_client_defs) . "};\n";
```

Comment:
Looks like these were the only real uses of `@all_server_defs` and `@all_client_defs`, so we can remove their remaining references too.